app/src/components/AnimalList.js

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+
+import AnimalCard from "./AnimalCard";
+
+const AnimalList = ({ animals }) => {
+  let cards = animals.map((animal) => (
+    <AnimalCard key={animal.id} {...animal} />
+  ));
+
+  return (
+    <>
+      <h1>Cards</h1>
+      <div
+        style={{
+          overflowY: "scroll",
+          maxHeight: "300px",
+          borderStyle: "solid",
+          borderColor: "white",
+        }}
```

Comment:
Would `useStyles` help here like you used elsewhere or is that tied to Material UI?

Being consistent is usually a good goal to aim for, but it may not be possible.

Reply:
@vegetabill do you know if it's possible to make a styles file and export them as needed? Or if it'd be even a good idea or if it'd overcomplicate things?

If it's plausible, I'm wondering if it'd be helpful for consistency?

```js
// reactStyling.js
import { makeStyles } from "@material-ui/core/styles";

export const cardsBox= {
          overflowY: "scroll",
          maxHeight: "300px",
          borderStyle: "solid",
          borderColor: "white",
        };

export const iconBtnStyle = makeStyles((theme) => ({
  button: {
    margin: theme.spacing(1),
  },
}));
....
```

and then import it as
```js
import { cardsBox } from './reactStyling';
...
<div
   style={cardsBox}
>
```

Reply:
I have no idea about this haha Im going to be honest! Will ask bill next week and update you Avery!